### PR TITLE
Feat: Perform manual scan of ID card photo

### PR DESCRIPTION
**Pull request by liordino**
feat: added a new state for the manual scan of id card photos on sintra

feat: pass the model as argument for compliance tiers

### CODE REVIEW

**Comment by liordino** [comment]
[Perform manual scan of ID card photo](https://trello.com/c/aMlBhw7n/197-perform-manual-scan-of-id-card-photo)

**Review by jsousa127 — APPROVED** [review]

File: ui/main.css [hunk 1]
```diff
@@ -472,6 +472,40 @@ button {
     .cash-out-color .filled-progress-button:active, .cash-out-color .filled-progress-button .active {
       color: #403c51; }
 
+.filled-progress-button-bg {
```
Comment (RafaelTaranto) [inline ui/main.css:475]: this has to be done on ui/css and build to this file

File: ui/js/app.js [hunk 2]
```diff
@@ -637,6 +637,7 @@ $(document).ready(function () {
   setupButton('facephoto-scan-failed-retry', 'retryFacephoto');
   setupButton('id-start-verification', 'permissionIdCompliance');
   setupButton('sms-start-verification', 'permissionSmsCompliance');
+  setupButton('ready-to-scan-id-card-photo', 'scanIdCardPhoto');
```
Comment (RafaelTaranto) [inline ui/js/app.js:640]: these changes have to be made on ui/src/app.js and build to this file

File: ui/start.html [hunk 3]
```diff
@@ -1171,6 +1171,42 @@ <h2 class="js-i18n primary-side-title xs-margin-bottom">Please verify your ident
         </section>
       </main>
     </section>
+    <section
```
Comment (RafaelTaranto) [inline ui/start.html:1174]: has to be changed in the ui/html folder and builded to this file

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Review by jsousa127 — APPROVED** [review]

**Review by RafaelTaranto — APPROVED** [review]

File: lib/brain.js [hunk 4]
```diff
@@ -580,6 +580,9 @@ Brain.prototype.transitionScreen = function transitionScreen () {
       case 'scanPhotoCard':
         appState = 'scan_id_photo'
         break
+      case 'scanPhotoCardSintra':
```
Comment (joshmh) [inline lib/brain.js:583]: I think it would be better if this was "scanPhotoCardManual".

File: lib/compliance/flows/id-card-photo.js [hunk 5]
```diff
@@ -11,7 +11,18 @@ const idPhotoMachine = xstate.Machine({
   strict: true,
   states: {
     idle: {
-      on: { START: 'scanPhotoCard' }
+      on: {
+        START: 'scanPhotoCard',
+        START_SINTRA: 'scanPhotoCardSintra'
```
Comment (joshmh) [inline lib/compliance/flows/id-card-photo.js:16]: This should also be START_MANUAL.

**Review by joshmh — CHANGES_REQUESTED** [review]

**Pull request closed without merging**